test/Microsoft.Health.Fhir.Tests.E2E/Microsoft.Health.Fhir.Tests.E2E.csproj

```diff
@@ -50,7 +50,27 @@
     </None>
   </ItemGroup>
   <Import Project="Microsoft.AspNetCore.Mvc.Testing.targets" />
+
+  <Target Name="VerifyExactSdkVersion" BeforeTargets="Build">
```

Comment:
Can/should any of this go in Build.props? Is it self-explanatory enough just to throw the error here?

Reply:
We don't want this to evaluate for every project. One project is enough. This seemed like a good one to me :). What would you add to the error message?

Reply:
I was just referring to only one project failing to build. In regards to the message, we could reference the download url maybe (https://dotnet.microsoft.com/download), but I think it's fine otherwise. Folks who download the code will probably see this sooner or later

Reply:
Updated the message